keyboards/dz60/keymaps/olligranlund_iso/keymap.c

```diff
@@ -19,20 +35,75 @@
 * `-----------------------------------------------------------------------------------------'
 */
 
+enum custom_keycodes {
+    EMOJI_DANCERS = SAFE_RANGE,
+	EMOJI_PERJANTAI,
+	EMOJI_THISISFINE,
+	EMOJI_KOVAAAJOA,
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+	switch (keycode) {
+    	case EMOJI_DANCERS:
+			if (record->event.pressed) {
+				// when keycode QMKBEST is pressed
+				SEND_STRING(">dancers>");
+			} else {} break;
```

Comment:
```suggestion
			}
			break;
```
Don't need the else blocks if they're empty.